packages/suite/src/components/suite/Settings/components/ActionColumn.tsx

```diff
@@ -29,6 +29,10 @@ const InputWrapper = styled.div`
         margin: 0px 0 10px;
         width: 100%;
     }
+
+    input {
```

Comment:
I see why you changed that. However, dropdown component is still 48px. So the solution is to increase height of buttons, which is 38px or decrease also dropdown's height. Please remove it from this PR, it is not related to ASCIII